packages/api/src/env.ts

```diff
@@ -7,8 +7,11 @@ export const env = setEnv({
   required: {
     nodeEnv: 'NODE_ENV',
     port: 'PORT',
+    baseURL: 'BASE_URL',
```

Comment:
I _think_ this is my opinion; thoughts?
```suggestion
    baseUrl: 'BASE_URL',
```